process/smartContract/process.go

```diff
@@ -172,12 +173,12 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 	sw.Stop("execute")
 	duration := sw.GetMeasurement("execute")
 
-	logFunc := log.Trace
 	if duration > executeDurationWarnThreshold {
-		logFunc = log.Warn
+		log.Warn(fmt.Sprintf("scProcessor.ExecuteSmartContractTransaction(): execution took > %s", duration), "sc", tx.GetRcvAddr(), "data", string(tx.GetData()), "duration", duration, "returnCode", returnCode, "err", err)
```

Comment:
I think we might have this as Debug, will evaluate later the maximum executeDurationWarnThreshold for a minimum spec node machine